examples/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
   cmake_minimum_required(VERSION 3.12)
   project(ArborXExamples CXX)
-  find_package(ArborX 1.2 REQUIRED)
+  find_package(ArborX 1.3 REQUIRED)
```

Comment:
Why does this one need to be updated?

Reply:
Because we don't want to worry that the examples work with the previous version of ArborX? For example, if start requiring Kokkos 3.5.

Reply:
Sure